wremnants/datasets/datasetDict_v9.py

```diff
@@ -24,8 +24,8 @@
     },
     'DYJetsToMuMuMass10to50PostVFP' : {
                    'filepaths' :
-                    ["{BASE_PATH}/BKGV9/DYJetsToMuMu_M-10to50_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos",
-                     "{BASE_PATH}/BKGV9/DYJetsToMuMu_M-10to50_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos_ext1"],
+                    ["{BASE_PATH}/{BKG_PATH_TAG}/DYJetsToMuMu_M-10to50_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos",
+                     "{BASE_PATH}/{BKG_PATH_TAG}/DYJetsToMuMu_M-10to50_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos_ext1"],
```

Comment:
The backgrounds should really follow the same naming convention as the signal, with the {NANO_PROD_TAG} especially.

Otherwise this will become unmanageable the minute any new NANOAOD has to be produced.

If this is too complicated to maintain together with support for the old background samples then I would just drop support for the old background samples now.

Reply:
I guess it is fine to remove support for old backgrounds. It also implies multiple modifications to remove any "if" statements to select custom branches depending on the NanoAOD, but in fact removing them will simplify the code.

I think I'll do as you say